### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by objorkman**
* Berkeley Mobile joins the dark side
Simulator Screen Shot - iPhone 13 Pro - 2021-11-03 at 17 37 02

### CODE REVIEW

File: berkeley-mobile/Tab Bar/TabBarController.swift [hunk 1]
```diff
@@ -37,6 +37,10 @@ class TabBarController: UITabBarController, UITabBarControllerDelegate {
         let attributes = [NSAttributedString.Key.font: Font.regular(11)]
         tabBarAppearance.setTitleTextAttributes(attributes, for: .normal)
         
+        if #available(iOS 15.0, *) {
+            tabBar.scrollEdgeAppearance = tabBar.standardAppearance
+        }
+
```
Comment (KevWho) [inline berkeley-mobile/Tab Bar/TabBarController.swift:43]: Is this needed? It seems the same as [line 33](https://github.com/asuc-octo/berkeley-mobile-ios/blob/73daeb2f312c93573999c792323a99fe135de7f3/berkeley-mobile/Tab%20Bar/TabBarController.swift#L33) above.

File: berkeley-mobile/Assets/Colors/Colors+Calendar.swift [hunk 2]
```diff
@@ -18,11 +18,19 @@ extension Color {
         }
         
         static var grayedText: UIColor {
-            return UIColor(displayP3Red: 193/255, green: 192/255, blue: 193/255, alpha: 1.0)
+            return UIColor.init { (trait) -> UIColor in
+                return trait.userInterfaceStyle == .dark ?
+                    UIColor(displayP3Red: 170/255, green: 170/255, blue: 170/255, alpha: 1.0) :
+                    UIColor(red: 98.0 / 255.0, green: 97.0 / 255.0, blue: 98.0 / 255.0, alpha: 1.0)
```
Comment (KevWho) [inline berkeley-mobile/Assets/Colors/Colors+Calendar.swift:24]: I guess we should probably stick to either `UIColor(displayP3Red: ...)` or `UIColor(red: ...)` for consistency.

File: berkeley-mobile.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1505,7 +1505,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 4HBETBULVA;
+				DEVELOPMENT_TEAM = K8FUZZ3FQV;
```
Comment (KevWho) [inline berkeley-mobile.xcodeproj/project.pbxproj:1508]: We should probably keep the old development team

**Review by KevWho — CHANGES_REQUESTED** [review]

**Review by KevWho — CHANGES_REQUESTED** [review]
This is a bit difficult to see (zoom in), but on the dining menus, the tableview background is white, so there's a sharp white rectangular around the dining items. Might be better to make the background for this table clear and have the cells be white for more contrast.

Screen Shot 2021-11-03 at 6 15 05 PM

**Review by huangshawn1 — CHANGES_REQUESTED** [review]
the table here also has a rectangular background, should make it transparent
IMG_2605

**Comment by huangshawn1** [comment]
looks like there's some issues with dark mode in study pact creation pages. I'm not in any StudyPact groups but we should also test that the group pages work
IMG_2607
IMG_2608

**Review by KevWho — APPROVED** [review]
LGTM!

**Review by huangshawn1 — APPROVED** [review]

**Pull request merged**